packages/ckeditor5-dev-release-tools/vitest.config.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig( {
+	plugins: [
+		executeInParallelVirtualModulePlugin()
+	],
+	test: {
+		setupFiles: [
+			'./tests/_utils/testsetup.js'
+		],
+		testTimeout: 10000,
+		mockReset: true,
+		restoreMocks: true,
+		include: [
+			'tests/**/*.js'
+		],
+		exclude: [
+			'./tests/_utils/**/*.js',
+			'./tests/test-fixtures/**/*.js'
+		],
+		coverage: {
+			provider: 'v8',
+			include: [
+				'lib/**'
+			],
+
+			reporter: [ 'text', 'json', 'html', 'lcov' ]
+		}
+	}
+} );
+
+function executeInParallelVirtualModulePlugin() {
```

Comment:
Could you explain how does it work and what are the benefits? Could it be used in other modules?

Reply:
A Vitest environment provides a fake module that can be imported from a `virtual:parallelworker-integration-module` path.

As a worker script (`parallelworker.js`) loads a callback to execute via an import (previously via `require()`), we need to provide something that would not throw an error. Due to different `cwd` depending on a trigger of tests (an npm script, an IDE integration, etc.), we need to predict an absolute path. While it's doable, it complicates things that do not have to.

Hence, I decided to create a fake module that is not used in source code but only in tests.